app/src/App/Controller/Administrator/Activate.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Controller\Administrator;
+
+use Demo\Domain\Service\Administrator\SetActiveInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+
+class Activate
+{
+    public function __construct(
+        private SetActiveInterface $setActive
+    ) {
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        /** @var int $administratorId */
+        $administratorId = $request->get('id');
```

Comment:
We should not assume that id exists nor that it's an integer (it will be a string actually)